src/main/webapp/css/business-casual.css

```diff
@@ -154,6 +154,10 @@ p {
   padding: 3rem;
   margin: 0.5rem;
   background-color: rgba(255, 255, 255, 0.85);
+  
+  /* NOTE: This container makes all child clickable, therefore make it pointer-events:none,
```

Comment:
I just realised this was in `business-casual.css`, would it be a better idea to move it to a different css file (maybe `user-page.css`)? I feel like changing the theme styling may not be the best idea because it may have unintended effects on other pages. #67 does add this to `main.css`, would it enough to have it there?

Just as I was testing, I found an example. This change prevents the links to user pages in the community page from being clickable since it sets pointer-events to none.

Reply:
#67 is a good example! I will move the pointer-events:none to user-page.css then!